### PR TITLE
Add documentation for mihonapp/mihon #420

### PR DESCRIPTION
**Pull request by sirlag**
Updated FAQ section for the upcoming updates screen as per inclusion in [#420](https://github.com/mihonapp/mihon/pull/420)

### CODE REVIEW

File: website/src/docs/faq/upcoming.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Upcoming
+Frequently Asked Questions about Upcoming Updates
+
+## What is this Calendar?
+The Upcoming Updates page does *not* consult with external trackers, publishers, or otherwise claim that a chapter update will be released on a particular day.
+
+What it does do is allow users of Mihon to visually plot their upcoming smart updates, as well as get a feel for when new chapters are *expected* to be released.
+
+## What are Smart Updates?
+In order to reduce strain on our partners and save user bandwidth, Mihon tries to estimate when chapter updates will be available, and only once that time has past include that series in automatic updates.
```
Comment (loocool2) [inline website/src/docs/faq/upcoming.md:16]:
```suggestion
In order to reduce strain on sources and save user bandwidth, Mihon tries to estimate when chapter updates will be available, and only once that time has passed, include that series in automatic updates.
```
You shouldn't refer to sources as partners as we don't have any relationship with sources and in my opinion referring to them as partners implies one. Also a little grammar change.

File: website/src/docs/faq/upcoming.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Why are my chapter updates on the wrong day?
+Smart updates determine time of chapter updates based on the time between currently released chapters. Sources that provide incorrect update dates, periods of hiatus, or bursts of short chapters may result in the estimate seeming off. As more chapters are released, this difference should normalize.
```
Comment (loocool2) [inline website/src/docs/faq/upcoming.md:19]:
```suggestion
Smart updates determine time of chapter updates based on the time between currently released chapters. Sources that provide incorrect update dates, periods of hiatus, or update in batches may result in the estimate seeming off. As more chapters are released, the schedule should become more accurate.
```
batch updates is probably a better way to refer this. Latter change should be a little more clear on what difference is.

**Review by loocool2 — CHANGES_REQUESTED** [review]
Looks pretty good but there are a few places that I think should be changed to be clearer.

**Comment by sirlag** [comment]
Seems like good changes, thanks @loocool2

**Comment by Soitora** [comment]
Maybe place the page at a subpage to Updates
`/docs/faq/updates/upcoming` rather than `/docs/faq/upcoming`

And then rename the page to just Upcoming

Rest looks fine to me, albeit maybe a bit too technically answered

**Comment by sirlag** [comment]
@Soitora Only issue with that suggestion is we don't have a `/updates` currently. We could create one and put the info from the `!smartupdates` command from the discord into it.

**Comment by Soitora** [comment]
> @Soitora Only issue with that suggestion is we don't have a `/updates` currently. We could create one and put the info from the `!smartupdates` command from the discord into it.

I can look into adding it tomorrow or something, unless you wanna make a simple PR for it

**Review by Soitora — APPROVED** [review]

**Pull request merged**